### DISCUSSION TITLE
Tohyo UX

### DISCUSSION THREAD

**Discussion by faermanj** [CaravanaCloud/tohyo#3] [Ideas]
Hello friends :)

This thread is to define how the system is going to work. Meaning the first step we're going to try to let organizations (syndicates, schools, companies, ...) create secure voting ballots, with digital and physical attendance.

Here are a few initial ideas; please post yours freely so that we can elaborate upon them.

- A web-based admin to create pools
- A mobile/web app to let people vote
- A physical voting device / urn with extended features
- Leverage blockchain so that it's crypto-safe

What do you think?

**Comment by carlosgc** [comment]
Great ideas!

Today, I started thinking about this project and began drafting some ideas. I tried to brainstorm use cases for this solution, from a online public pool to a large voting system run by a syndicate, like the one I participated in this year.

For documentation and as a complement, I'll put here the ideas that came to mind when I thought about the project:

# Overview

- Allow creating polls.
- Allows you to register the options that will receive votes. (People, proposals or anything that a decision was wanted by vote).
- Allow voter registration.
- Allow voters to vote.
- Present voting results.

## Possible voting options

- Voters
	- Allow voting without prior registration of voters.
	- Allow prior registration of voters.
	- Allow checking whether a voter is able to vote.

- Vote control
	- Allow voting without controlling the number of votes per voter.
	- Allow only one vote per voter.
	- Allow voters to modify their own vote.

## Implementation ideas

Voting via the internet and/or mobile devices.
Store votes on blockchain.
Print votes to enable verification.
